utils/kubernetes/deployment.go

```diff
@@ -111,6 +111,21 @@ func GetContainerByName(name string, podSpec *v1.PodSpec) (*v1.Container, int) {
 	return nil, -1
 }
 
+// GetContainerPortByName returns a pointer to the ContainerPort within the given Container.
+// If none found, returns nil.
+// It also returns the position where the container port was found, -1 if none.
+func GetContainerPortByName(name string, container *v1.Container) (*v1.ContainerPort, int) {
+	if container == nil {
```

Comment:
should this instead be `if container != nil {` and just `return nil, -1` once at the end of the function?

Reply:
I think we have here different programming style, and really no criteria used around the project in general.
Let me explain, the intention here is that you don't have to wait to the end of the function to know what happens, instead, as soon you know the container is null, you don't need to read the remainder of the code, you know directly what the result should be.  As said, it correspond to a programming styling, all variants are good.
For this particular file, we had already a function GetContainerByName https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/1eb14dfcd37705ebcb7c60342063bf3f438532a0/utils/kubernetes/deployment.go#L102, that uses this style.
I you don't mind, I'd prefer to follow same patter for a similar function.

Reply:
definitely understand the approach, but i should have noticed this pattern being used in the existing functions... my bad

Reply:
yeah, we don't have really a standardized adoption, but yeah, in this case ha had this situation.